packages/icons/src/library/cloud.js

```diff
@@ -4,8 +4,8 @@
 import { SVG, Path } from '@wordpress/primitives';
 
 const cloud = (
-	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="-2 -2 24 24">
```

Comment:
`-2 -2` was meant to offset some icons that are designed on the 20px grid, assuming the new icon isn't.

Reply:
correct, all the new "g2" icons are on a 24px grid.